tests/infer/test_elbo_mapdata.py

```diff
@@ -18,7 +18,7 @@
 @pytest.mark.stage("integration", "integration_batch_1")
 @pytest.mark.init(rng_seed=161)
 @pytest.mark.parametrize("batch_size", [3, 5, 7, 8, None])
-@pytest.mark.parametrize("map_type", ["tensor", "list"])
+@pytest.mark.parametrize("map_type", ["iarange", "irange", "range"])
```

Comment:
@neerajprad We're actually running more tests in integration_batch_1, so that might be part of the slowdown.